torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -614,7 +633,7 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
     }
   }
 
-  KernelArgumentHolder kernel_arguments;
+  KernelArgumentHolder kernel_arguments(options_.index_mode);
```

Comment:
options_ has been stored in `FusionExecutor` when `compileFusion`. 
https://github.com/csarofeen/pytorch/blob/e4b37b857966ada532c41da401b290f018242bad/torch/csrc/jit/codegen/cuda/executor.cpp#L157